content/en/docs/collector/getting-started.md

```diff
@@ -171,12 +171,12 @@ locally to a file. Data is sent to the container and the container scrapes its o
 Prometheus metrics.
 
 ```console
-$ git clone git@github.com:open-telemetry/opentelemetry-collector-contrib.git; \
+$ git clone https://github.com/open-telemetry/opentelemetry-collector-contrib.git; \
     cd opentelemetry-collector-contrib/examples/demo; \
-    go build client/main.go; ./client/main & pid1="$!"; \
-    go build server/main.go; ./server/main & pid2="$!"; \
+    go build -o client/main client/main.go; ./client/main & pid1="$\!"; \
```

Comment:
The build is failing because the Go module is located within the client directory, not the demo:

```
$ go build -o client/main client/main.go
client/main.go:32:2: missing go.sum entry for module providing package go.opentelemetry.io/otel/exporters/otlp/otlpmetric; to add:
	go mod download go.opentelemetry.io/otel/exporters/otlp
client/main.go:33:2: missing go.sum entry for module providing package go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetricgrpc; to add:
	go mod download go.opentelemetry.io/otel/exporters/otlp
client/main.go:34:2: missing go.sum entry for module providing package go.opentelemetry.io/otel/exporters/otlp/otlptrace; to add:
	go mod download go.opentelemetry.io/otel/exporters/otlp
client/main.go:35:2: missing go.sum entry for module providing package go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc; to add:
	go mod download go.opentelemetry.io/otel/exporters/otlp

$ cd client/
$ go build -o main main.go
$ ls -l main
-rwxr-xr-x. 1 jpkroehling wheel 15415056 Mar 17 13:14 main
```

Reply:
Great catch!!! I really appreciate you noticing this @jpkrohling! I'll send up another commit that fixes this.